packages/datagateway-dataview/src/views/landing/isis/isisInvestigationLanding.component.test.tsx

```diff
@@ -453,4 +453,61 @@ describe('ISIS Investigation Landing page', () => {
       'John Smith et al; 2019: Test 1, doi_constants.publisher.name, https://doi.org/doi 1'
     );
   });
+
+  it('copies data citation to clipboard', () => {
+    // Mock the clipboard object
+    const testWriteText = jest.fn();
+    Object.assign(navigator, {
+      clipboard: {
+        writeText: testWriteText,
+      },
+    });
+
+    const testStore = mockStore({
+      ...state,
+      dgcommon: {
+        ...state.dgcommon,
+        data: [
+          {
+            ...state.dgcommon.data[0],
+            investigationUsers: [investigationUser[0]],
+          },
+        ],
+      },
+    });
+    const wrapper = mount(
+      <Provider store={testStore}>
+        <MemoryRouter>
+          <ISISInvestigationLanding
+            instrumentId="4"
+            instrumentChildId="5"
+            investigationId="1"
+            studyHierarchy={false}
+          />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    expect(
+      wrapper
+        .find('[aria-label="landing-investigation-citation"]')
+        .first()
+        .text()
+    ).toEqual(
+      'John Smith; 2019: Test 1, doi_constants.publisher.name, https://doi.org/doi 1'
+    );
+
+    wrapper
+      .find('#landing-investigation-copy-citation')
+      .first()
+      .simulate('click');
+
+    expect(navigator.clipboard.writeText).toHaveBeenCalledWith(
```

Comment:
Sorry Goel, I know it's annoying, you just missed one reference! In line with the other one, change this to:
`expect(testWriteText).toHaveBeenCalledWith(
      'John Smith; 2019: Title 1, doi_constants.publisher.name, https://doi.org/study pid'
);`

Reply:
Not at all! 😄 I didn't see this, thanks Sam, that's brilliant!